test/benchmarks/png_funcs.h

```diff
@@ -0,0 +1,146 @@
+#pragma once
+
+#include <stdlib.h>
+#include <stdint.h>
+#include <string.h>
+
+#define IMWIDTH 1024
+#define IMHEIGHT 1024
+
+extern "C" {
+    #include <png.h>
```

Comment:
It should be `#  include <png.h>` like in the other files with two spaces between # and include and no tab at the beginning?

Reply:
This is honestly one of the more oddball style choices I've seen but I can make it consistent.